.cirrus.yml

```diff
@@ -20,8 +20,9 @@ env:
     ####
     #### Cache-image names to test with
     ###
-    PRIOR_RHEL_CACHE_IMAGE_NAME: "rhel-7-conmon-10ec3343"
-    FEDORA_CACHE_IMAGE_NAME: "fedora-29-conmon-10ec3343"
+    # Note rhel tests are disabled because crio rpms haven't been found for them
+    # PRIOR_RHEL_CACHE_IMAGE_NAME: 'rhel-7-conmon-75ea13be'
```

Comment:
This is fine for testing, but please just clobber all of `*RHEL*` found anywhere in this file and under `contrib/cirrus/`